{{ cookiecutter.name }}/Resources/Strings.swift

```diff
@@ -0,0 +1,36 @@
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
```

Comment:
This file is generated by swiftgen command. So why should we embed it, instead of generating via swiftgen?

Reply:
Yes, you are right. These files are redundant.